### PR TITLE
Fix docs about item_selected and item_focused . Fixes #25273

### LINKED ISSUE #25273: OptionButton item_selected signal sends an index even though IDs are defined
Godot 3.1 beta 2 and ab843b16984d8b62a1100d9b77d8b0dcdc4252bb (by looking at the code)

From the Q&A thread https://godotengine.org/qa/39689/optionbutton-passing-an-id-with-add_item

`OptionButton` has an `item_selected` signal, with a signature and description indicating that it "sends the ID of the item that was selected. If IDs were not used, the index is sent instead".

However, the value received is never the ID, it's always the index, which tremendously reduces the usability of IDs.

Test project:
[OptionButtonAlwaysSendsIndex.zip](https://github.com/godotengine/godot/files/2789445/OptionButtonAlwaysSendsIndex.zip)

### PR DESCRIPTION
**Pull request by azagaya**
…nals in optionbutton

Description in docs about item_selected and item_focused signals in optionbutton is fixed to match the real behaviour. Also, get_item_index function is added.

*Bugsquad edit:* Fixes #25273.

### CODE REVIEW

**Comment by azagaya** [comment]
Wow, why could appveyor fail?

File: doc/classes/OptionButton.xml [hunk 1]
```diff
@@ -74,6 +74,15 @@
 				Return the ID of the item at index "idx".
 			</description>
 		</method>
+                <method name="get_item_index" qualifiers="const">
+                        <return type="int">
+                        </return>
+                        <argument index="0" name="id" type="int">
+                        </argument>
+                        <description>
+                                Return the index of the item with id "id".
```
Comment (akien-mga) [inline doc/classes/OptionButton.xml]: Use `[code]id[/code]` for the argument name. And you can avoid repeating "id" with e.g. "with the given [code]id[/code]".

**Comment by akien-mga** [comment]
Please squash commits/drop the second commit.

**Comment by akien-mga** [comment]
See [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase) for instructions on rebasing your branch.

**Review by azagaya — COMMENTED** [review]
All god now i think.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**